microsoft-edge/devtools-guide-chromium/whats-new/2021/04/devtools.md

```diff
@@ -180,7 +180,7 @@ To manually update to version 1.1.7, navigate to [Update an extension manually](
 
 ### Visualize CSS scroll-snap
 
-You may now toggle the `scroll-snap` badge in the **Elements** tool to inspect the CSS scroll-snap alignment.  When an HTML element on your webpage has `scroll-snap-type` applied to it, a `scroll-snap` badge displays next to it in the **Elements** tool.  Choose the badge to turn on (or off) the display of a scroll-snap overlay on the webpage.  To review an example webpage, navigate to [Scroll Snap Demo](https://microsoft-edge-chromium-devtools.glitch.me/css-dbg-stories/css-scroll-snap.html).  In the example, dots display on snap edges.  The scroll port has a solid outline while the snap items have dash outlines.  The scroll padding is filled in green while the scroll margin is filled in orange.  To review the history of this feature in the Chromium open-source project, navigate to Issue [862450](https://crbug.com/862450).
+You may now toggle the `scroll-snap` badge in the **Elements** tool to inspect the CSS scroll-snap alignment.  When an HTML element on your webpage has `scroll-snap-type` applied to it, a `scroll-snap` badge displays next to it in the **Elements** tool.  Choose the badge to turn on (or off) the display of a scroll-snap overlay on the webpage.  To review an example webpage, navigate to [Scroll Snap Demo](https://mathiasbynens.github.io/css-dbg-stories/css-scroll-snap.html).  In the example, dots display on snap edges.  The scroll port has a solid outline while the snap items have dash outlines.  The scroll padding is filled in green while the scroll margin is filled in orange.  To review the history of this feature in the Chromium open-source project, navigate to Issue [862450](https://crbug.com/862450).
```

Comment:
I didn't move this demo from glitch to github. It had already been copied from Mathias' excellent demo page. I don't see a reason not to link directly to it instead.
Especially because this is part of the upstream chromium announcements too.